posix/subsystem/src/nl-socket.cpp

```
@@ -110,9 +110,7 @@ struct OpenFile : File {
 		using namespace protocols::fs;
 		if(logSockets)
 			std::cout << "posix: Recv from socket \e[1;34m" << structName() << "\e[0m" << std::endl;
-		assert(!flags);
```

We should probably warn about flags we don't support, e.g. by printing the flags `if(!(flags & ~(MSG_DONTWAIT | MSG_CMSG_CLOEXEC)))`, as is done in similar fashion elsewhere, too. Unsupported flags should be silently ignored AFAICT (looking at the man page and POSIX).